.travis.yml

```diff
@@ -25,7 +25,6 @@ before_script:
   - Xvfb :99 -screen 0 1024x768x24 -extension RANDR &
 
 script:
-  - npm run clean
```

Comment:
I never took the time to figure out what the problem exactly was, but without cleaning first S3 was definitely getting builds uploaded that were not the current one that was done on travis.